python/ray/experimental/data/read_api.py

```diff
@@ -17,12 +17,30 @@
 def autoinit_ray(f):
     def wrapped(*a, **kw):
         if not ray.is_initialized():
-            ray.client.connect()
+            ray.client().connect()
         return f(*a, **kw)
 
     return wrapped
 
 
+def _parse_paths(paths: Union[str, List[str]]
+                 ) -> Tuple["pyarrow.fs.FileSystem", Union[str, List[str]]]:
+    from pyarrow import fs
+    if isinstance(paths, str):
+        return fs.FileSystem.from_uri(paths)
+    else:
+        if len(paths) == 0:
+            raise ValueError("No data provided")
+        parsed_results = [fs.FileSystem.from_uri(path) for path in paths]
+        filesystem = parsed_results[0][0]
+        paths = []
+        for (tmp_fs, path) in parsed_results:
```

Comment:
Nit: Parentheses not necessary.
```suggestion
        for tmp_fs, path in parsed_results:
```